tests/integration/dcr/dcr_test.go

```diff
@@ -0,0 +1,533 @@
+/*
```

Comment:
Shall we move this into `tests/integration/identity/oauth2/dcr` package?
Can you also rename `identity` folder to `oauth`?